docs/basics/usage.md

```diff
@@ -23,36 +23,36 @@ Counterparty database files are by default named `counterparty.[testnet.]db` and
 * Linux: `~/.local/share/counterparty`
 * Windows: `%APPDATA%\Roaming\Counterparty\counterparty`
 
-All configurable parameters in the configuration file can also be passed as arguments to the `counterpart-server` command. Use `counterparty-server --help` to see the list of these options.
+All configurable parameters in the configuration file can also be passed as arguments to the `counterpart-core` command. Use `counterparty-core --help` to see the list of these options.
```

Comment:
sorry, misunderstanding! I think we should keep the command as `counterparty-server